src/integrationTest/java/com/pubnub/api/integration/HeartbeatIntegrationTest.java

```diff
@@ -44,12 +45,13 @@ protected void onAfter() {
     }
 
     @Test
-    public void testStateWithHeartbeat() {
+    public void testStateWithHeartbeat() throws PubNubException {
```

Comment:
Yes, I don't like this inconvenience neither.  I can modify UserId constructor to throw RuntimeException instead of PubNubException that is checked exception.